ModList/UI/Markdig/UnityRenderer.cs

```diff
@@ -159,57 +171,68 @@ private IEnumerable<RectTransform> RenderParagraph(ParagraphBlock para)
             {
                 var inline = RenderInline(para.Inline, ParagraphFontSize);
                 foreach (var child in inline)
+                {
                     child.SetParent(transform, false);
+                }
             }
 
             AfterObjectRendered?.Invoke(para, transform.gameObject);
 
-            return Helpers.SingleEnumerable(transform).Append(Spacer(1.5f));
+            return transform.SingleEnumerable().Append(Spacer(1.5f));
```

Comment:
The reason I was using this method explicitly rather than as an extension was because I realized that it was more readable not as an extension method. I must have forgotten to make it not an extension method.